bin/tree-filter.php

```diff
@@ -115,11 +115,10 @@
 }
 
 // Test directory files
-copy($assetDir . '/test-files/gitignore', 'test/.gitignore');
 file_put_contents(
     'test/Bootstrap.php',
     str_replace('{COMPONENT}', $normalized, file_get_contents($assetDir . '/test-files/Bootstrap.php'))
 );
-copy($testConfigDist, 'test/TestConfiguration.php.dist');
-copy($testConfigTravis, 'test/TestConfiguration.php.travis');
+copy($testConfigDist, 'TestConfiguration.php.dist');
+copy($testConfigTravis, 'TestConfiguration.php.travis');
```

Comment:
No. I did this for a reason. I wanted the test assets (bootstrap, test configuration) in the `/test` root, which is why I pushed the component tests one level down. I really don't want the bootstrap or test configuration files under the root; `phpunit.xml.dist` makes sense there to simplify running the tests, but it acts like the other dot-files in that regard.